src/server.rs

```diff
@@ -29,22 +29,22 @@ use crate::{
         mainchain::{
             create_sidechain_proposal_response, get_bmm_h_star_commitment_response,
             get_ctip_response::Ctip, get_sidechain_proposals_response::SidechainProposal,
-            get_sidechains_response::SidechainInfo, server::ValidatorService,
-            wallet_service_server::WalletService, BroadcastWithdrawalBundleRequest,
+            get_sidechains_response::SidechainInfo, send_transaction_request::FeeRate,
```

Comment:
This is in the cargo fmt commit: would be great if you could run cargo fmt on each commit instead of a separate formatting commit. Bad git hygiene! You can do a `git rebase -i origin/master` where you stop on each commit, run `cargo fmt` and then `commit --amend --no-edit`